packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx

```diff
@@ -192,6 +195,26 @@ const SidebarNavigation: React.FC<SidebarNavigationProps> = () => {
               })}
           </SiteNavNestList>
         </SiteNavItem>
+        <SiteNavItem>
+          <SiteNavButton
+            onClick={() => setIconSystemOpen(!iconSystemOpen)}
+            isOpen={iconSystemOpen}
+            aria-expanded={iconSystemOpen}
+          >
+            Icon System
+            <SiteNavAnchorArrow isOpen={iconSystemOpen} />
+          </SiteNavButton>
+          <SiteNavNestList isOpen={iconSystemOpen}>
+            <SiteNavItem>
+              <SiteNavAnchor to={SidebarCategoryRoutes.ICON_SYSTEM}>Usage Guidelines</SiteNavAnchor>
+            </SiteNavItem>
+            <SiteNavItem>
+              <SiteNavAnchor to={`${SidebarCategoryRoutes.ICON_SYSTEM}/how-to-add-an-icon`}>
+                How to add an icon
```

Comment:
Seems like there's some automatic title casing going on in the sidebar, which makes it a little weird that the H1 is sentence case, and the sidebar item is title case.

Not important for this PR but noting it for the future.

Reply:
Should I leave it, remove title casing on the sidebars, or add it to h1s?

Reply:
Do nothing for now. It'd be ideal to remove the title casing from the sidebars, but I'm worried we'll end up with some unintentional casing there.